guides/doc-Administering_Red_Hat_Satellite/topics/Monitoring_Capsules.adoc

```diff
@@ -31,15 +31,15 @@ Click the number to view the details of associated hosts.
 *How much storage space is available on {SmartProxyServer}?*:: The amount of storage space occupied by the Pulp content in `/var/lib/pulp` is displayed.
 Also the remaining storage space available on the {SmartProxy} can be ascertained.
 
-[[sect-Documentation-Administering_Red_Hat_Satellite-Monitoring_Services]]
+[[sect-Administering-Monitoring_Services]]
 ==== Monitoring Services
 
 Navigate to *Infrastructure* > *{SmartProxies}* and click the name of the selected {SmartProxy}.
 At the *Services* tab, you can find basic information on {SmartProxy} services, such as the list of DNS domains, or the number of Pulp workers.
 The appearance of the page depends on what services are enabled on {SmartProxyServer}.
-Services providing more detailed status information can have dedicated tabs at the {SmartProxy} page (see xref:sect-Documentation-Administering_{Project_Link}-Monitoring_Puppet[]).
+Services providing more detailed status information can have dedicated tabs at the {SmartProxy} page (see xref:sect-Documentation-Administering-Monitoring_Puppet[]).
```

Comment:
This link is broken. It appears as: 

the Smart Proxy page (see [sect-Documentation-Administering-Monitoring_Puppet]).

Reply:
Rebased to master & fixed.